src/nvme/mi.h

```diff
@@ -367,6 +367,34 @@ struct nvme_mi_admin_resp_hdr {
 	__le32	cdw0, cdw1, cdw3;
 } __attribute__((packed));
 
+/**
+ * struct nvme_mi_service_req_hdr - servicing command request header.
+ * @hdr: Generic MI message header
+ * @opcode: servicing command opcode (using &enum nvme_service_opcode)
+ * @tag: flag - be used to sent and return
+ * @cpsp: Control Primitive Specific Parameter
+ */
+struct nvme_mi_service_req_hdr {
+	struct nvme_mi_msg_hdr hdr;
+	__u8	opcode;
+	__u8	tag;
+	__le16	cpsp;
+} __attribute((packed));
+
+/** struct nvme_mi_service_resp_hdr - servicing command response header.
+ * @hdr: Generic MI message header
+ * @status: Servicing command status (using &enum nvme_service_status)
+ * @tag: flag - Be used to sent and return
+ * @cpsr: Control Primitive Specific Response
+ */
+
+struct nvme_mi_service_resp_hdr {
+	struct nvme_mi_msg_hdr hdr;
+	__u8	status;
+	__u8	tag;
+	__le16	cpsr;
+} __attribute((packed));
```

Comment:
And same here, this is the entire response, not just the header.

Reply:
done